### ISSUE TITLE
Rule Proposal: Warn if there are repetitive locators in a page object

### ISSUE THREAD

**Issue by alecxe** [alecxe/eslint-plugin-protractor#25]
Example of a violation:

``` js
var MyPage = function () {
   this.grids = element.all(by.css(".mygrid"));
   this.firstGrid = element.all(by.css(".mygrid")).first();
}
```

Valid version:

``` js
var MyPage = function () {
   this.grids = element.all(by.css(".mygrid"));
   this.firstGrid = this.grids.first();
}
```

**Comment by alecxe** [comment]
Closed by 04da60c29d5fdf6887cabbc733527ff8de663096.
